ooni/errors.py

```diff
@@ -26,6 +27,8 @@
     (CancelledError, 'task_timed_out'),
     (gaierror, 'address_family_not_supported_error'),
     (DNSLookupError, 'dns_lookup_error'),
+    (DNSNameError, 'dns_name_error'),
+    (DNSServerError, 'dns_server_failure'),
```

Comment:
Okay. You should probably also add those error strings to [data-formats/df-000-base.md](https://github.com/TheTorProject/ooni-spec/blob/master/data-formats/df-000-base.md) in TheTorProject/ooni-spec#49, right?